todo_app/app.py

```diff
@@ -1,7 +1,7 @@
-from flask import Flask, app, render_template, url_for
+from flask import Flask, render_template, url_for
 from werkzeug.utils import redirect
 from todo_app.data.session_items import *
-from todo_app.flask_config import Config
+#from todo_app.flask_config import Config
```

Comment:
Now that this is confirmed working, let's delete it rather than commenting it out